clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/ClusteredStore.java

```diff
@@ -582,15 +651,19 @@ public void initStore(final Store<?, ?> resource) {
       clusteredStore.storeProxy.addInvalidationListener(new ServerStoreProxy.InvalidationListener() {
         @Override
         public void onInvalidateHash(long hash) {
+          Enum result = StoreOperationOutcomes.EvictionOutcome.SUCCESS;
+          clusteredStore.evictionObserver.begin();
```

Comment:
I talked to Mathieu about creating some clustered eviction tests in :clustered:integration-test but at the moment there is an issue (Terracotta-OSS/terracotta-core#297) preventing use of the management module.  Once this is fixed I can proceed.